server/app/views/admin/programs/ProgramApplicationView.java

```diff
@@ -151,4 +164,33 @@ private DivTag renderAnswer(long programId, AnswerData answerData) {
                 .withClasses(
                     Styles.FLEX_AUTO, Styles.TEXT_RIGHT, Styles.FONT_LIGHT, Styles.TEXT_XS));
   }
+
+  private static SelectTag renderStatusOptionsDropdown(StatusDefinitions statusDefinitions) {
+    SelectTag dropdownTag =
```

Comment:
From a usability standpoint, we may want to add a label to the dropdown so that the user knows that this corresponds to a status (or add help text or some other affordance). I'd check with Raha on this. If we went the label route, a benefit would be that we could update the selector in the browser tests to select by that instead.

Reply:
Do you mean using SelectWithLabel ?

What's the main benefit of label here for screen readers?

Reply:
Not necessarily SelectWithLabel, since that tends to display pretty large and might not be suitable from a styling standpoint here.

My main concern is that someone might not know what the dropdown actually represents. Having a label indicating that it's for an application status may aid in understanding. It likely also will be a bit preferable from an a11y standpoint as well.